code/game/objects/structures/guillotine.dm

```diff
@@ -248,20 +248,12 @@
 
 	return ..()
 
-/obj/structure/guillotine/wrench_act(mob/living/user, obj/item/I)
+/obj/structure/guillotine/wrench_act(mob/living/user, obj/item/tool)
 	. = ..()
-	if (current_action)
```

Comment:
is the current action var used elsewhere? what's going on here?

Reply:
its used for when you're pulling the lever/picking it back up, and preventing it from being unwrenched while someone's interacting with the lever via can_be_unfasten_wrench()

Reply:
shouldn't it still be set then so people can't lever it while it's being unwrenched? why remove that bit?

Reply:
it does already with ``can_be_unfasten_wrench(mob/user, silent)``

Reply:
excuse me I had that backwards, doesn't this allow you to start an unwrench, and then pull the lever? also why does `GUILLOTINE_ACTION_WRENCH` still exist if you don't use it anymore